pages/pipelines/writing_build_scripts.md.erb

```diff
@@ -49,7 +49,7 @@ For a full list of options, see the [Bash Reference Manual](https://www.gnu.org/
 
 <section class="Docs__troubleshooting-note">
   <h3>Unbound variable errors</h3>
-  <p>Note that while enabling the <code>u</code> option is generally a good default to use for all build scripts, it can cause some tools like <a href="https://rvm.io">rvm</a> to fail with “unbound variable” errors. If you encounter errors, you can either remove <code>u</code> from the list of options, or you can surround the command causing the error with <code>set +u</code> and <code>set -u</code> to temporarily disable the option.</p>
+  <p>Note that while enabling the <code>u</code> option is generally a good default to use for all build scripts, it can cause some tools like <a href="https://rvm.io">rvm</a> to fail with “unbound variable” errors. If you encounter errors, you can either remove <code>u</code> from the list of options, or you can surround the command causing the error with <code>set +u</code> and <code>set -u</code> to temporarily disable the option. In some cases, you may need to run the tool itself wrapped in <code>set +u</code> and <code>set -u</code> For example: <code>set +u; rvm xxx; set -u</code></p>
```

Comment:
```suggestion
  <p>Note that while enabling the <code>u</code> option is generally a good default to use for all build scripts, it can cause some tools like <a href="https://rvm.io">rvm</a> to fail with “unbound variable” errors. If you encounter errors, you can either remove <code>u</code> from the list of options, or run the tool causing the error wrapped in <code>set +u</code> and <code>set -u</code> to remove the option just for that command. For example: <code>set +u; rvm xxx; set -u</code>.</p>
```